src/main/java/common/Constants.java

```diff
@@ -5,4 +5,6 @@
     public static final String URL = "http://www.railway2.somee.com/";
     public static final String USER = "autouser@gmail.com";
     public static final String PASSWORD = "123456789";
+    public static final Integer ADD_DAY = 10;
```

Comment:
what is add day?

Reply:
ADD_DAY to use for method getDayFromToday

Reply:
I fixed it. Please help me check it. Thanks so much